### PR TITLE
Edit React state

### LINKED ISSUE #457: [Edit] React : State
### Type of Edit

Add new content (definitions, codeblocks, etc.)

### Description

Now that hooks have been the default way of using state since 2019 I propose that we add a part showing how use state with `useState()` since react beginners will be using it.

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by yangc95**
### Description

Closes #457  

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@yangc95, I looked this over and didn't see any changes I'd suggest. I'd pas it on to another review or merge as-is. (@Dusch4593 , @KTom101 )

File: content/react/concepts/state/state.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ The `state` object is where we store data in a component that is expected to cha
 
 Props are passed down by parent components, whereas state is created and maintained by the component itself.
```
Comment (Dusch4593) [inline content/react/concepts/state/state.md:18]:
```suggestion
[Props](https://www.codecademy.com/resources/docs/react/props) are passed down by parent components, whereas state is created and maintained by the component itself.
```

File: content/react/concepts/state/state.md [hunk 2]
```diff
@@ -76,9 +78,11 @@ class Car extends React.Component {
 
 It will look like:
 
-## My First Car
+```html
+<h1>My First Car</h1>
 
-It is a white Chevrolet Malibu from 1998. 🚙
+<p>It is a white Chevrolet Malibu from 1998. 🚙</p>
```
Comment (Dusch4593) [inline content/react/concepts/state/state.md:84]:
```suggestion
<p>It is a white Chevrolet Malibu made in 1998. 🚙</p>
```

File: content/react/concepts/state/state.md [hunk 3]
```diff
@@ -76,9 +78,11 @@ class Car extends React.Component {
...
+```
 
 ## Caution
```
Comment (Dusch4593) [inline content/react/concepts/state/state.md:87]: Can we edit the text below this header and remove all "you"-voices?

File: content/react/concepts/state/state.md [hunk 4]
```diff
@@ -76,9 +78,11 @@ class Car extends React.Component {
 
 It will look like:
```
Comment (Dusch4593) [inline content/react/concepts/state/state.md:79]:
```suggestion
It will look like this:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@yangc95 Left a few comments/suggestions for you. Solid edits! 👍

**Comment by yangc95** [comment]
> @yangc95 Left a few comments/suggestions for you. Solid edits! 👍

Thanks @Dusch4593 ! MMade some changes based on your suggestions/comments~

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍 Ready for merge?

**Comment by yangc95** [comment]
Edits should appear here: https://www.codecademy.com/resources/docs/react/state

ty! @SSwiniarski @Dusch4593

**Pull request merged**